### PR TITLE
Release 8: Add showing error message when test was already finished

### PR DESCRIPTION
**Pull request by mBeym**
Mantis Ticket: https://mantis.ilias.de/view.php?id=36512

- Clicking on the `Finish Test`, `Next question` or `Previous question` button will now show an error message if the user has no unfinished passes currently (meaning it was already finished from somewhere/someone else.

### CODE REVIEW

**Comment by dsstrassner** [comment]
Hi @matthiaskunkel 


please have a look at the new lang_var and the introduced text. This fix should remove 1 failing Test case 😉

@mbecker-databay / @kergomard please wait for the feedback and approval from @matthiaskunkel

**Comment by mBeym** [comment]
@dsstrassner lang fixed :)

File: lang/ilias_de.lang
Comment (matthiaskunkel) [inline lang/ilias_de.lang:1375]: Testdurchlauf, nicht Test Durchlauf

File: lang/ilias_en.lang
Comment (matthiaskunkel) [inline lang/ilias_en.lang:1375]: According to our English expert Chris Potter, the term 'pass' should only be used for a successful test run (I still would like to streamline this until 8.0 stable). The use of 'pass' in this lang var shows the problem. The test run is not even completed but called pass. That's confusing for a native speaker. Better would be: 'Your current test run is no longer valid. <br>It was probably completed already or finished by a tutor.' And neither 'test run' nor 'Tutor' needs capital letters. 'Tutor' in capital letters would refer to the role 'Tutor', while 'tutor' is a general term for those people who are tutoring...

**Review by matthiaskunkel — APPROVED** [review]
Thanks @mbeym for the fixes. Can be merged now.

**Pull request merged**
